pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/tagger/utils"
 	"github.com/DataDog/datadog-agent/pkg/util/containers"
 	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/kubelet"
+
+	"k8s.io/kubernetes/third_party/forked/golang/expansion"
```

Comment:
do we already have a dep on it?

Reply:
Yes. That’s why I didn’t have to add it in the list of things that we vendor.
It’s the module that the kubelet uses to implement variable interpolation.
As we vendor kubernetes, we already vendor this one.

Reply:
I initially wanted to use [os.Expand](https://golang.org/pkg/os/#Expand), but is expands `${…}` whereas the syntax for variable expansion in Kubernetes is `$(…)` so I looked at what the kubelet was using and I decided to use the same module.